.travis.yml

```diff
@@ -1,14 +1,34 @@
-sudo: false # enable container based builds (can't use sudo)
+sudo: required
+
 language: cpp
-script: make tarball
 
 compiler:
     - gcc
-    # - clang
 
 os:
-    - linux
     - osx
+    - linux
+
+env:
+    global:
+    - DEPS_UBUNTU="g++ cmake libboost-regex-dev xfslibs-dev libssl-dev default-jdk python-dev libfuse-dev libkrb5-dev"
+    - DEPS_CENTOS="gcc-c++ make cmake boost-devel krb5-devel xfsprogs-devel openssl-devel python-devel fuse-devel java-openjdk java-devel libuuid-devel"
```

Comment:
Can you put these in the same order in both systems as much as possible? Also, you need to add git and puppet so that we can get access to facter.